lib/linguist/languages.yml

```diff
@@ -121,6 +121,14 @@ APL:
   codemirror_mode: apl
   codemirror_mime_type: text/apl
   language_id: 6
+ASL:
+  type: programming
+  ace_mode: text
+  extensions:
+  - ".asl"
+  - ".dsl"
+  tm_scope: source.asl
+  language_id: 124996147
```

Comment:
I note you've not chosen a colour. This is fine, but it does mean the language will get the dull default of `#cccccc`.

Reply:
it's appropriate given how boring language is